backend/api/src/error.rs

```diff
@@ -597,6 +598,12 @@ impl Into<actix_web::Error> for JigCloneDraft {
             )
             .into(),
 
+            Self::IncompleteModules => BasicError::with_message(
```

Comment:
@johnnynotsolucky we might need to disable this check (temporarily?) since this call is also used in curation and they wont be able to curate jigs that are incomplete and have been published already

Reply:
@MendyBerger I included a migration which marks existing JIGs with content as complete in the database. Are there still JIGs which are published and incomplete?